docs/connecting-to-a-database.md

```diff
@@ -0,0 +1,97 @@
+# Connecting to a database
+
+> ⚠️ **Warning**: Particularly if you're connecting to a production database,
+> proceed with caution and look to pair with someone else.
+
+Using the AWS console, you can connect via a bastion host to the:
+
+- [PostgreSQL database](#connecting-to-the-postgresql-rds-postgresql-database)
+- [MongoDB database](#connecting-to-the-mongodb-documentdb-database)
+
+## Prerequisites
+
+- Access to the relevant AWS account for the environment
+
+## Environment information
+
+| Environment | AWS account       | Systems Manager key | EC2 jumpbox                      |
+|-------------|-------------------|---------------------|----------------------------------|
+| Staging     | StagingAPIs       | staging             | RDS jump box-Platform APIs (new) |
+| Production  | Mosaic-Production | mosaic-prod         | bastion-rds-jump-box-prod        |
+
+## Connecting to the PostgreSQL (RDS PostgreSQL) database
+
+1. Within relevant AWS account, go to **Systems Manager** → **Parameter Store**
+2. Search for **/social-care-case-viewer-api/\<environment-key\>/postgres**
+3. In a new tab, go to **Session Manager** under **Node Management** in **Systems Manager**
+4. Click on **Start session**
+5. Select the appropriate EC2 jumpbox and click on **Start session**
+6. Using the parameters from Parameter Store in your other tab, connect to the PostgreSQL database by running:
+
+```
+$ psql --host=<hostname> --port=<port> --username=<username> --dbname=social_care
+```
```

Comment:
We could mention here that you would be prompted for a password and where to get it depending on the environment—e.g. secrets manager for the production, parameter store for the staging account, etc.

Reply:
I update the doc with hopefully better clarity, let me know what you think of https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/323/commits/16ce6d4fb8fc60695343293d1fa46829e3b77eff.